### PR TITLE
fix for issue 335. For PO028, a policy not supported in a profile now results in an error.

### PR DESCRIPTION
**Pull request by DinoChiesa**
The PO028 plugin now issues an Error, not a Warning, when encountering a policy that is not supported in the given profile.
This addresses issue #335 .

### CODE REVIEW

File: lib/package/plugins/policyMetaData.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
   "OAuthV1" :{"indications":["oauthv1","oauth","oa","accesstoken","verify"], "profiles":["apigee"] },
   "OAuthV2" :{"indications":["oauthv2","oauth","oa","accesstoken","verify"]},
   "OASValidation" :{"indications":["oas","oasValidation","^oas$"]},
-  "ParseDialogRequest":{"indications":["parsedialogrequest","dialogflow","pdr","^parsedialogrequest$"], "profiles":["apigeex"] },
+  "ParseDialogflowRequest":{"indications":["parsedialogrequest","dialogflow","pdr","^parsedialogrequest$"], "profiles":["apigeex"] },
```
Comment (ssvaidyanathan) [inline lib/package/plugins/policyMetaData.json:41]: `{"indications":["parsedialogrequest","dialogflow","pdr","^parsedialogrequest$"], "profiles":["apigeex"] }`

should we rather do
`{"indications":["parsedialogflowrequest","dialogflow","pdr","^parsedialogflowrequest$"], "profiles":["apigeex"] }`

File: lib/package/plugins/policyMetaData.json [hunk 2]
```diff
@@ -48,7 +48,7 @@
   "ResponseCache" :{"indications":["responsecache","rc","cache","^responsecache$"]},
   "Script" :{"indications":["script","scr"]},
   "ServiceCallout" :{"indications":["callout","sc"]},
-  "SetDialogResponse" :{"indications":["setdialogresponse","dialogflow","sdr","^setdialogresponse$"], "profiles":["apigeex"] },
+  "SetDialogflowResponse" :{"indications":["setdialogresponse","dialogflow","sdr","^setdialogresponse$"], "profiles":["apigeex"] },
```
Comment (ssvaidyanathan) [inline lib/package/plugins/policyMetaData.json:51]: `{"indications":["setdialogresponse","dialogflow","sdr","^setdialogresponse$"], "profiles":["apigeex"] }`

should we rather do
`{"indications":["setdialogflowresponse","dialogflow","sdr","^setdialogflowresponse$"], "profiles":["apigeex"] }`

**Review by ssvaidyanathan — CHANGES_REQUESTED** [review]

File: lib/package/plugins/policyMetaData.json (hunk 1 above)
Comment (DinoChiesa) [reply lib/package/plugins/policyMetaData.json]: done

File: lib/package/plugins/policyMetaData.json (hunk 2 above)
Comment (DinoChiesa) [reply lib/package/plugins/policyMetaData.json]: done

**Comment by DinoChiesa** [comment]
PTAL

**Review by ssvaidyanathan — APPROVED** [review]

**Pull request merged**